person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  # contructor
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getters and setters
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- [ ] [_Optional_] You should consider adding the attribute accessibilities at the start of the class, even before the initialize since it's a norm followed by most of the Ruby development community.